package.json

```diff
@@ -7,6 +7,7 @@
     "start": "node index.js",
     "develop": "node index.js --develop",
     "db-migrate": "dropdb bitballs --if-exists && createdb bitballs && ./node_modules/db-migrate/bin/db-migrate up",
+    "generate-docs": "documentjs -d",
```

Comment:
normally we call this `document` in the DoneJS generators:  https://github.com/donejs/chat/blob/master/package.json#L15

but keep the `-d`